packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -3576,6 +3576,56 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+    delete:
+      tags:
+        - eservices
+      summary: Deletes an e-service
+      operationId: deleteEService
+      description: Deletes the specified E-Service
+      parameters:
+        - name: eserviceId
+          in: path
+          description: the eservice id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "204":
+          description: EService deleted
```

Consistency with the rest of the M2M spec
```suggestion
          description: E-Service deleted
```